docs/GettingStartedDocs/best_practices_secure_enclaves.md

```diff
@@ -0,0 +1,257 @@
+# Best Practices for Keeping Enclaves Secure
+Use this guide to apply secure patterns and avoid common mistakes that put the security of enclave applications at risk.
+
+1.  [Best Practices for Interface Custom Marshaling ](#best-practices-for-interface-custom-marshaling)
```

Comment:
This can be build as table to include the sub items that are listed under each of the best practices

Reply:
Done